### PR TITLE
chore: fix copyright banner

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Loong <loong.dai@intel.com>

### CODE REVIEW

**Comment by daixiang0** [comment]
I will run `make update-common` after https://github.com/istio/common-files/pull/717 merged.

File: proto/xds.proto [hunk 1]
```diff
@@ -7,166 +21,172 @@ package envoy.service.discovery.v3;
...
-    // A developer-facing error message, which should be in English. Any
-    // user-facing error message should be localized and sent in the
-    // [google.rpc.Status.details][google.rpc.Status.details] field, or localized by the client.
-    string message = 2;
-
-    // A list of messages that carry the error details.  There is a common set of
-    // message types for APIs to use.
-    // google.protobuf.Any
-    repeated google.protobuf.Any details = 3;
+  // The status code, which should be an enum value of
```
Comment (howardjohn) [inline proto/xds.proto:27]: This shouldn't change...?

File: proto/xds.proto [hunk 2]
```diff
@@ -1,3 +1,17 @@
+// Copyright Istio Authors
```
Comment (howardjohn) [inline proto/xds.proto:1]: This is not copyright Istio Authors, its copy+pasted from Envoy

File: proto/xds.proto (hunk 1 above)
Comment (daixiang0) [reply proto/xds.proto:27]: Yeah, it is about proto format, let me revert.

File: proto/xds.proto (hunk 2 above)
Comment (daixiang0) [reply proto/xds.proto:1]: Sure, let me fix it.

Comment (daixiang0) [reply proto/xds.proto]: I have reverted those changes, not sure what is the right repo and path they from, so keep it as before.

Maybe we can introduce a new target like `update-xds` in another PR.

Comment (howardjohn) [reply proto/xds.proto]: We don't want to update, it's pruned by hand to remove unused fields to remove bloat

**Review by howardjohn — APPROVED** [review]

**Pull request merged**